cli/dcoscli/marathon/main.py

```diff
@@ -11,6 +11,7 @@
     dcos marathon app show [--app-version=<app-version>] <app-id>
     dcos marathon app start [--force] <app-id> [<instances>]
     dcos marathon app stop [--force] <app-id>
+    dcos marathon app kill [--scale] [--host=<host>] <app-id>
     dcos marathon app update [--force] <app-id> [<properties>...]
     dcos marathon app version list [--max-count=<max-count>] <app-id>
     dcos marathon deployment list [--json <app-id>]
```

Comment:
Can we also add the new arguments to the docstring? https://github.com/mesosphere/dcos-cli/blob/master/cli/dcoscli/marathon/main.py#L30-L101